lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -381,6 +439,35 @@ module Resources
         },
         {
           'id': 'CI-97982588-1-0',
+          'extension': [
+            {
+              'valuePeriod': {
+                'start': '2016-12-19T16:44:25.000Z',
+                'end': '2022-05-01T16:56:56.000Z'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableconcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'cod': '159',
+                    'display': 'Sister',
+                    'userSelected': true
+                  },
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+                    'code': 'SIS',
+                    'display': 'sister',
+                    'userSelected': false
+                  }
+                ],
+                'text': 'Sister'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+            }
+          ],
           'coding': [
             {
               'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/40',
```

Comment:
code set 40 relations should no longer be in the realationship.coding

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL454
 and other places